Examples/Swift/LineStyleLayerExample.swift

```diff
@@ -28,8 +28,17 @@ class LineStyleLayerExample_Swift: UIViewController, MGLMapViewDelegate {
     func loadGeoJson() {
         DispatchQueue.global().async {
             // Get the path for example.geojson in the app’s bundle.
-            guard let jsonUrl = Bundle.main.url(forResource: "example", withExtension: "geojson") else { return }
-            guard let jsonData = try? Data(contentsOf: jsonUrl) else { return }
+            guard let jsonUrl = Bundle.main.url(forResource: "example", withExtension: "geojson") else {
+                preconditionFailure("Failed to load local GeoJSON file")
+            }
+            
+            guard let jsonData = try? Data(contentsOf: jsonUrl) else {
+                let alert = UIAlertController(title: "Error", message: "Failed to decode GeoJSON data", preferredStyle: .alert)
```

Comment:
An alert here seems out of place — let’s stick to failing hard and telling the developer about it. Our audience for these isn’t end-users so much as developers, so I think we can err on the less user-friendly side.